src/core/utilities/hash.test.ts

```diff
@@ -0,0 +1,10 @@
+import { hash } from "./hash";
+
+describe("hash", () => {
+  describe("hash", () => {
```

Comment:
Don't think you need this second describe.

Reply:
Fixed